src/main/java/nurseybook/logic/commands/AddCommand.java

```diff
@@ -68,6 +69,7 @@ public CommandResult execute(Model model) throws CommandException {
         }
 
         model.addElderly(toAdd);
+        model.updateFilteredElderlyList(PREDICATE_SHOW_ALL_ELDERLIES);
```

Comment:
I thought that if we were to add an elderly or a new task, the user would want to see it in the list, so I chose to display all.

Reply:
This complements with how whenever we add an elderly or task, the list display changes to whichever type of addition they made

Reply:
To clarify, previously, if we were to add an elderly on a filtered elderly list, the list will continue to display what is filtered instead of all the elderly in NurseyBook?

Reply:
Yep! I felt this might not be the best if the user was finding Colins and decided to add an Aaron- then they would not be able to see Aaron.